activestorage/app/models/active_storage/variant.rb

```diff
@@ -53,7 +53,7 @@
 # * {ImageProcessing::Vips}[https://github.com/janko-m/image_processing/blob/master/doc/vips.md#methods]
 # * {ruby-vips reference}[http://www.rubydoc.info/gems/ruby-vips/Vips/Image]
 class ActiveStorage::Variant
-  WEB_IMAGE_CONTENT_TYPES = %w[ image/png image/jpeg image/jpg image/gif ]
+  WEB_IMAGE_CONTENT_TYPES = %w[ image/png image/jpeg image/jpg image/gif image/tiff ]
```

Comment:
Given that TIFF isn’t supported by Chrome or Firefox, I don’t think we can avoid converting TIFF variants to PNG.

Reply:
@georgeclaghorn done